src/components/Captures/CaptureTable.js

```diff
@@ -258,26 +259,35 @@ const CaptureTable = () => {
           </TableRow>
         </TableHead>
         <TableBody data-testid="captures-table-body">
-          {captures &&
-            captures.map((capture) => (
-              <TableRow
-                key={capture.id}
-                onClick={createToggleDrawerHandler(capture.id)}
-                className={classes.tableRow}
-              >
-                {columns.map(({ attr, renderer }) => (
-                  <TableCell key={attr}>
-                    {formatCell(
-                      capture,
-                      speciesLookup,
-                      captureTagLookup[capture.id] || [],
-                      attr,
-                      renderer
-                    )}
-                  </TableCell>
-                ))}
-              </TableRow>
-            ))}
+          {captures.length === 0 ? (
```

Comment:
Unfortunately, this isn't quite enough. There may be occasions when zero captures match the filters, so we want to show an empty list.
Instead, we will need to add an `isLoading` boolean flag to CapturesContext that tracks whether a Captures load is in progress.

Reply:
This should be `{isLoading ? (`